rpc/amqp-rpc.go

```diff
@@ -239,47 +261,186 @@ type RPCResponse struct {
 	Error     RPCError `json:"error,omitempty"`
 }
 
-// Start starts the AMQP-RPC server running in a separate thread.
-// There is currently no Stop() method.
-func (rpc *AmqpRPCServer) Start() (err error) {
-	msgs, err := amqpSubscribe(rpc.channel, rpc.serverQueue, rpc.log)
+// AmqpChannel sets a AMQP connection up using SSL if configuration is provided
+func AmqpChannel(conf cmd.Config) (*amqp.Channel, error) {
```

Comment:
Please verb-ify this method name, e.g., `EstablishAmqpChannel`.  Also, there's a [general stylistic problem](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) here with `Amqp` in variable/method names, but that's probably better for a separate PR, e.g., #556.